packages/vapor/scss/components/menu.scss

```diff
@@ -1,13 +1,9 @@
 .dropdown.mod-menu {
-    &.header-section {
-        &:hover {
-            background-color: var(--header-section-hover-color);
-        }
-
-        &.open {
-            background-color: var(--header-section-hover-color);
-        }
+    &.header-section:hover,
+    &.header-section.open {
+        background-color: #191d3d;
```

Comment:
why hard coded? (which color is it? xD)

Reply:
https://github.com/coveo/admin-ui/blob/master/packages/v2/stylesheets/rebranding/_colors.scss#L82
This color is out of the palette, and it's only used here in one place. It has been put in admin-ui temporarily.